pipelines/tekton/aiedge-e2e/tasks/retrieve-build-image-info.task.yaml

```diff
@@ -40,7 +46,25 @@ spec:
       echo ;
       oc get -n $(params.namespace) -o jsonpath='{.image.dockerImageMetadata.Config.Labels.io\.buildah\.version}' imagestreamtag/$(params.model-name):$(params.model-version) | tee $(results.internal-image-buildah-version.path) ;
       echo ;
-      echo -n "$(params.target-registry-url):$(params.model-version)-$(params.pipeline-run-uid)" | tee $(results.target-image-url.path) ;
+      echo -n "$@" | tee $(results.target-image-tag-references.path) ;
+  - name: build-urls-txt
+    image: registry.access.redhat.com/ubi9/ubi-micro
+    args:
+      - "$(params.target-image-tag-references[*])"
+    script: |
+      #!/usr/bin/env bash
+
+      set -Eeuo pipefail
+
+      export URLTXT=$(workspaces.images-url.path)/url.txt
```

Comment:
skopeo-copy task looks for this file in its workspace if the source and destination parameters are empty. This is what allows pushing to more than one tag from the single taskrun.

Reply:
Maybe add that as a comment above the export line?